### DISCUSSION TITLE
Language Policy

### DISCUSSION THREAD

**Discussion by crazywoola** [langgenius/dify#3118] [General]
To facilitate clear and effective communication, all discussions, comments, documentation, and pull requests in this project should be conducted in English. This ensures that all contributors can participate and collaborate effectively.

为了促进清晰有效的沟通，本项目中的所有讨论、评论、文档和pull请求都应以英语进行。这确保了所有贡献者都能有效地参与和协作。

**Comment by Javad572** [comment]
> To facilitate clear and effective communication, all discussions, comments, documentation, and pull requests in this project should be conducted in English. This ensures that all contributors can participate and collaborate effectively.
> 
> 
> 
> 为了促进清晰有效的沟通，本项目中的所有讨论、评论、文档和pull请求都应以英语进行。这确保了所有贡献者都能有效地参与和协作。

**Reply by Javad572** [reply]
Xl

**Reply by Javad572** [reply]
> > To facilitate clear and effective communication, all discussions, comments, documentation, and pull requests in this project should be conducted in English. This ensures that all contributors can participate and collaborate effectively.
> > 
> > 
> > 
> > 为了促进清晰有效的沟通，本项目中的所有讨论、评论、文档和pull请求都应以英语进行。这确保了所有贡献者都能有效地参与和协作。
> 
>

**Comment by watt1010** [comment]
May I ask if the tool can return directly without undergoing large model conversion, like the tools in Langchain return_direct = True, If this function is available, it can convert messages into cards when connected

**Comment by BaxtiyorUrolov** [comment]
I want to add the Uzbek language. If you teach me how to add it, I will help you add the Uzbek language.
